### PR TITLE
Model formatting

### PR DESCRIPTION
**Pull request by KCNilssen**
Cleans up game a bit. Let me know if this is better and if it needs more but is in the right direction. 

Formats other 'models'

### CODE REVIEW

File: mlbstatsapi/models/game/livedata/plays/play/playrunner/attributes.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+from typing import Union, Optional
+from dataclasses import dataclass
+
+from mlbstatsapi.models.people import Person
+
+@dataclass
+class RunnerCreditsPosition:
```
Comment (Mattsface) [inline mlbstatsapi/models/game/livedata/plays/play/playrunner/attributes.py:7]: Have you seen this anywhere else? This has the same attributes as PrimaryPosition. I feel we should just rename it to Position and use the class here and for Person, Player, or where ever else.

Comment (Mattsface) [reply mlbstatsapi/models/game/livedata/plays/play/playrunner/attributes.py:7]: We don't have to address this now, but we should take note.

File: mlbstatsapi/models/schedules/schedule.py [hunk 2]
```diff
@@ -61,8 +74,8 @@ def get_games_with_status(self, abstractGameState = None,
                         gameIds.append(game.gamePk)
             return gameIds
 
-    def get_games_inProgress(self):
+    def get_games_inProgress(self) -> List[int]:
         return self.get_games_with_status(abstractGameState='Live')
 
-    def get_games_finnished(self):
+    def get_games_finnished(self) -> List[int]:
```
Comment (Mattsface) [inline mlbstatsapi/models/schedules/schedule.py:80]: finished* :)

**Review by Mattsface — APPROVED** [review]
LGTM good work

I would fix that "finished" typo though ;)

File: mlbstatsapi/models/game/livedata/plays/play/playrunner/attributes.py (hunk 1 above)
Comment (Mattsface) [reply mlbstatsapi/models/game/livedata/plays/play/playrunner/attributes.py:7]: Feel free to resolve this your self

Comment (KCNilssen) [reply mlbstatsapi/models/game/livedata/plays/play/playrunner/attributes.py:7]: Good catch Matt! Changed PrimaryPosition in people.attributes to Position and got rid of RunnerCreditsPosition.

File: mlbstatsapi/models/schedules/schedule.py (hunk 2 above)
Comment (KCNilssen) [reply mlbstatsapi/models/schedules/schedule.py:80]: No, I want to specifically get games that are Finnish. Good catch again mate haha

**Pull request merged**